MoltenVKShaderConverter/MoltenVKShaderConverterTool/MoltenVKShaderConverterTool.cpp

```diff
@@ -317,10 +213,6 @@ void MoltenVKShaderConverterTool::showUsage() {
 	log("                       source code files. The dirPath may be omitted to use");
 	log("                       the current working directory.");
 	log("  -r                 - (when using -d) Process directories recursively.");
-	log("  -gi [\"glslInFile\"] - Indicates that GLSL shader code should be input.");
```

Comment:
In the help text output by `showUsage()`, the description intro section (just above here) still documents GLSL usage. Please remove them to avoid user confusion.